api/go.mod

```diff
@@ -16,6 +16,7 @@ require (
 
 require (
 	github.com/fxamacker/cbor/v2 v2.7.0 // indirect
+	github.com/hashicorp/go-version v1.7.0 // indirect
```

Comment:
Do you understand where this dependency is coming from?
Does the `api` module has any dependency on procdiscovery?